libs/client-sdk/tests/client_c_bindings_test.cpp

```diff
@@ -20,3 +20,41 @@ TEST(ClientBindings, MinimalInstantiation) {
 
     LDClientSDK_Free(sdk);
 }
+
+void ListenerFunction(char const* flag_key,
+                      LDValue new_value,
+                      LDValue old_value,
+                      bool deleted,
+                      void* user_data) {
+}
+
+// This test registers a listener. It doesn't use the listener, but it
+// will at least ensure 1.) Compilation, and 2.) Allow sanitizers to run.
+TEST(ClientBindings, RegisterFlagListener) {
```

Comment:
When we do TestData for the server possibly we can add something similar to the client. It would make legitimate testing of these much simpler. 

On the C++ side it is better tested because we can directly make the updater.

Reply:
I did test this a number of ways in the hello-c app.